contracts/v0.8/DataCapAPI.sol

```diff
@@ -53,17 +53,17 @@ library DataCapAPI {
     function symbol() internal returns (string memory) {
         bytes memory raw_request = new bytes(0);
 
-        bytes memory raw_response = Actor.call(DataCapTypes.SymbolMethodNum, DataCapTypes.ActorCode, raw_request);
+        bytes memory raw_response = Actor.call(DataCapTypes.SymbolMethodNum, DataCapTypes.ActorCode, raw_request, Misc.NONE_CODEC);
 
         bytes memory result = Actor.readRespData(raw_response);
 
         return result.deserializeString();
     }
 
-    function totalSupply() internal returns (int256) {
-        bytes memory raw_request = BytesCBOR.serializeNull();
+    function totalSupply() public returns (int256) {
```

Comment:
Tests fail because it should be internal. Otherwise the contract won't be embeddable